package.json

```diff
@@ -60,7 +60,7 @@
   },
   "homepage": "https://github.com/asciidoctor/asciidoctor.js",
   "dependencies": {
-    "opal-runtime": "0.11.0-integration8"
+    "opal-runtime": "1.0.2"
```

Comment:
This makes me very happy to see ;)

Reply:
It's not really a 1.0.0 even though Opal is considering to release a 1.0.0 after 0.11. But the version was confusing so I've opted to use semantic versioning (with a compatibility matrix table).

Reply:
But I like that it's actually based on a commit in the upstream history. That's progress.

Reply:
Indeed no need to use an integration branch anymore 🎉